diff --git a/node_modules/@tryghost/kg-markdown-html-renderer/lib/markdown-html-renderer.js b/node_modules/@tryghost/kg-markdown-html-renderer/lib/markdown-html-renderer.js
index 5f0bc1b..f7b049f 100644
--- a/node_modules/@tryghost/kg-markdown-html-renderer/lib/markdown-html-renderer.js
+++ b/node_modules/@tryghost/kg-markdown-html-renderer/lib/markdown-html-renderer.js
@@ -1,6 +1,8 @@
 const MarkdownIt = require('markdown-it');
+const {createHighlightFunc, katexPlugin, useContainer} = require('@traptitech/traq-markdown-it');
 
 const markdownIt = new MarkdownIt({
+    highlight: createHighlightFunc('blog-code'),
     html: true,
     breaks: true,
     linkify: true
@@ -47,6 +49,11 @@ markdownIt.linkify.set({
     fuzzyLink: false
 });
 
+markdownIt.use(katexPlugin, {
+    output: 'html'
+});
+useContainer(markdownIt);
+
 module.exports = {
     render: function (markdown) {
         return markdownIt.render(markdown);
